src/navigation/Routes.android.js

```diff
@@ -322,6 +280,43 @@ function BSNavigator() {
         name={Routes.EXPANDED_ASSET_SHEET_POOLS}
         options={expandedPresetWithSmallGestureResponseDistance}
       />
+      {profilesEnabled && (
+        <>
+          <BSStack.Screen
+            component={ENSConfirmRegisterSheet}
+            name={Routes.ENS_CONFIRM_REGISTER_SHEET}
+          />
+          <BSStack.Screen
+            component={ProfileSheet}
+            name={Routes.PROFILE_SHEET}
+          />
+          <BSStack.Screen
+            component={RegisterENSNavigator}
+            name={Routes.REGISTER_ENS_NAVIGATOR}
+          />
+          <BSStack.Screen
+            component={ENSAdditionalRecordsSheet}
+            name={Routes.ENS_ADDITIONAL_RECORDS_SHEET}
+          />
+          <BSStack.Screen
+            component={SelectENSSheet}
+            name={Routes.SELECT_ENS_SHEET}
+          />
+          <BSStack.Screen
+            component={ProfileSheet}
+            name={Routes.PROFILE_PREVIEW_SHEET}
+          />
+          <BSStack.Screen
+            component={SelectUniqueTokenSheet}
+            name={Routes.SELECT_UNIQUE_TOKEN_SHEET}
+            options={{ ...bottomSheetPreset, height: '95%' }}
```

Comment:
could we standardize this height? like `height - SOME_PADDING` or  `height - StatusBar.currentHeight`
Because this indentation varies on different devices